packages/toast/src/use-toast.tsx

```diff
@@ -189,16 +189,19 @@ export function createStandaloneToast({
  */
 export function useToast(options?: UseToastOptions) {
   const { theme, setColorMode, toggleColorMode, colorMode } = useChakra()
+
+  const [toastOptions] = React.useState(options)
```

Comment:
Why not `useRef`? Also do we allow the default options to be changed after rendered?